demo/runtime/wasm/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "demo-runtime"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+substrate-codec = { path = "../../../substrate/codec", default-features = false }
+substrate-runtime-std = { path = "../../../substrate/runtime-std", default-features = false }
+substrate-runtime-io = { path = "../../../substrate/runtime-io", default-features = false }
+substrate-runtime-support = { path = "../../../substrate/runtime-support", default-features = false }
+substrate-primitives = { path = "../../../substrate/primitives", default-features = false }
+demo-primitives = { path = "../../primitives", default-features = false }
+integer-sqrt = "0.1.0"
```

Comment:
Since `integer-sqrt` doesn't have `#![no_std]` it will implictly pull std and you will face an error "duplicate lang item `panic_fmt`".

Maybe it would be a good idea to put `./build.sh` in .travis.yml?

Reply:
`integer-sqrt` makes no use of the standard library. 

https://github.com/derekdreery/integer-sqrt-rs/blob/master/Cargo.toml#L11

Reply:
Indeed, just slapping `#![no_std]` worked out for me.

Reply:
yes, it's falsely categorized in the no-std group. although it doesn't use `std` explicitly it doesn't tell `rustc` not to link it.